green/config.py

```diff
@@ -628,7 +638,8 @@ def getConfig(  # pragma: no cover
 
     cwd = pathlib.Path.cwd()
     # Medium priority
-    for cfg_file in ("setup.cfg", ".green"):
+    config_files = [] if not supports_tomllib else ["pyproject.toml"]
```

Comment:
Let's set this up so that when we drop support for Python < 3.11, it's easy to understand how to remove the conditional handling here:

```suggestion
    config_files = ["pyproject.toml", "setup.cfg", ".green"]
    if not supports_tomllib:
        config_files.remove("pyproject.toml")
```